README.md

```diff
@@ -2,52 +2,50 @@
 <img src="doc/logo.png" width=256 align=right>
 
 >*OpenJsCad.org* is a more up-to-date [OpenJsCAD](http://joostn.github.com/OpenJsCad/) frontend where you can edit .jscad files either locally or online via JS editor (built-in).
-A few functions are available to make the transition from [OpenSCAD](http://openscad.org/) to OpenJSCAD easier ([OpenSCAD.jscad](https://github.com/Spiritdude/OpenSCAD.jscad) built-in),
+A few functions are available to make the transition from [OpenSCAD](http://openscad.org/) to OpenJSCAD easier ([OpenSCAD.jscad](https://github.com/jscad/OpenSCAD.jscad) built-in),
 as well CLI (command-line interface) for server-side computations with NodeJS.
 
-[![GitHub version](https://badge.fury.io/gh/Spiritdude%2FOpenJSCAD.org.svg)](https://badge.fury.io/gh/Spiritdude%2FOpenJSCAD.org)
+[![GitHub version](https://badge.fury.io/gh/jscad%2FOpenJSCAD.org.svg)](https://badge.fury.io/gh/jscad%2FOpenJSCAD.org)
 [![experimental](http://badges.github.io/stability-badges/dist/stable.svg)](http://github.com/badges/stability-badges)
-[![Build Status](https://travis-ci.org/Spiritdude/OpenJSCAD.org.svg)](https://travis-ci.org/Spiritdude/OpenJSCAD.org)
-[![Dependency Status](https://david-dm.org/Spiritdude/OpenJSCAD.org.svg)](https://david-dm.org/Spiritdude/OpenJSCAD.org)
-[![devDependency Status](https://david-dm.org/Spiritdude/OpenJSCAD.org/dev-status.svg)](https://david-dm.org/Spiritdude/OpenJSCAD.org#info=devDependencies)
-
-
+[![Build Status](https://travis-ci.org/jscad/OpenJSCAD.org.svg)](https://travis-ci.org/jscad/OpenJSCAD.org)
+[![Dependency Status](https://david-dm.org/jscad/OpenJSCAD.org.svg)](https://david-dm.org/jscad/OpenJSCAD.org)
+[![devDependency Status](https://david-dm.org/jscad/OpenJSCAD.org/dev-status.svg)](https://david-dm.org/jscad/OpenJSCAD.org#info=devDependencies)
 
 ## Table of Contents
 
 - [Installation](#installation)
-- [History](#History)
+- [Contribute](#contribute)
+- [Documentation](#Documentation)
 - [Community](#Community)
+- [License](#license)
 - [Todo](#todo)
-- [Documentation](#Documentation)
+- [History](#History)
 - [Screenshots](#Screenshots)
-- [Contribute](#contribute)
-- [License](#license)
 - [See also](#Also)
 
 ## Installation
 
 ### Immediate Use (no installation)
 
-go to *[OpenJSCAD.org](http://openjscad.org)* (Google Chrome, Firefox)
+Go to *[OpenJSCAD.org](http://openjscad.org)* (Tested browsers include Chrome, Firefox, Opera, Safari)
 
-### Local Web Use
+### Use within a Web Site
 
 ```
-% git clone https://github.com/Spiritdude/OpenJSCAD.org
-% cd OpenJSCAD.org
-% make install
+cd base-directory-of-website
```

Comment:
This is actually not 100% true, we deliver the distributables dist/index dist/opts etc , so you do not need NPM or anything except a base server with the html, js , css files